app/js/app-utilities.js

```diff
@@ -2073,15 +2075,52 @@ appUtilities.animateToOtherEnd = function(edge, mouse_position) {
   var source_loc = Math.pow((mouse_position.x - source_position.x), 2) + Math.pow((mouse_position.y - source_position.y), 2);
   var target_loc = Math.pow((mouse_position.x - target_position.x), 2) + Math.pow((mouse_position.y - target_position.y), 2);
   
-  var pan_target = (source_loc < target_loc) ? target_node : source_node;
+  // Animation direction
+  var source_to_target = (source_loc < target_loc) ? true : false;
+
+  var bend_points = [];
+  if(bend_instance !== undefined && bend_instance.getSegmentPoints(edge) !== undefined){
+    bend_points = bend_instance.getSegmentPoints(edge);
+  }
+  
+  // Read bend points according to animation direction
+  if(!source_to_target){
```

Comment:
`bend_points.reverse();`

Reply:
the result is not the exact reverse of the array. it reverses (x,y) pairs.

`ex. [1, 2, 3, 4] -> [3, 4, 1, 2]`